go.sum

```diff
@@ -1,1191 +1,48 @@
-4d63.com/gochecknoglobals v0.1.0/go.mod h1:wfdC5ZjKSPr7CybKEcgJhUOgeAQW1+7WcyK8OvUilfo=
-bitbucket.org/creachadair/shell v0.0.6/go.mod h1:8Qqi/cYk7vPnsOePHroKXDJYmb5x7ENhtiFtfZq8K+M=
-cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Oh wow, I guess one of the packages we use cleaned up their dependencies a bunch? :)

(noticed because of PR size)
<img width="153" alt="Screenshot 2023-12-10 at 19 12 57" src="https://github.com/tailscale/mkctr/assets/1687799/73e3c51b-c721-4f1b-968e-77c77bbbcf12">

Reply:
No, I am actually not sure how we had this, I think it just somehow had a bunch of extra deps, maybe by accident? If I downgrade the containerregistry back to v0.9.0 and run `go mod tidy` again, I don't get envoy proxy, grpc and all these other deps that